client/state/action-types.js

```diff
@@ -42,6 +42,7 @@ export const POSTS_REQUEST_SUCCESS = 'POSTS_REQUEST_SUCCESS';
 export const PUBLICIZE_CONNECTIONS_RECEIVE = 'PUBLICIZE_CONNECTIONS_RECEIVE';
 export const PUBLICIZE_CONNECTIONS_REQUEST = 'PUBLICIZE_CONNECTIONS_REQUEST';
 export const PUBLICIZE_CONNECTIONS_REQUEST_FAILURE = 'PUBLICIZE_CONNECTIONS_REQUEST_FAILURE';
+export const READER_FULLPOST_SET_VISIBILITY = 'READER_FULLPOST_SET_VISIBILITY';
```

Comment:
Thoughts on splitting this into two actions, `READER_FULLPOST_SHOW` and `READER_FULLPOST_HIDE` ? Is there an advantage to being able to set any old value?